apps/condo/domains/banking/schema/BankAccount.js

```diff
@@ -179,6 +200,26 @@ const BankAccount = new GQLListSchema('BankAccount', {
             type: Json,
             isRequired: false,
         },
+
+        classificationCode: {
+            schemaDoc: 'Budget classification code, used for state-funded organizations',
+            type: Text,
+            isRequired: false,
+            hooks: {
+                validateInput: ({ context, existingItem, resolvedData }) => {
```

Comment:
Any test for this validation?

Reply:
yes, in validation test itself (apps/condo/domains/banking/utils/validate/classificationCode.utils.spec.js)